src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,16 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
+    @Size(min = 8)
```

Comment:
check size in the custom validator
![image](https://github.com/mate-academy/jv-spring-security/assets/20326858/3e63f779-09d1-47cf-8103-14d99de4c61b)